shell/platform/windows/flutter_windows.cc

```diff
@@ -29,6 +30,33 @@
 
 static_assert(FLUTTER_ENGINE_VERSION == 1, "");
 
+// Attempts to load AOT data from the given path, which must be absolute and
+// non-empty. Logs and returns nullptr on failure.
+UniqueAotDataPtr LoadAotData(std::filesystem::path aot_data_path) {
+  if (aot_data_path.empty()) {
+    std::cerr
+        << "Attempted to load AOT data, but no aot_library_path was provided."
+        << std::endl;
+    return nullptr;
+  }
+  if (!std::filesystem::exists(aot_data_path)) {
+    std::cerr << "Can't load AOT data from " << aot_data_path.u8string()
+              << "; no such file." << std::endl;
+    return nullptr;
+  }
+  std::string path_string = aot_data_path.u8string();
+  FlutterEngineAOTDataSource source;
```

Comment:
Nit: Zero out C structs please. `= {}` here and `= nullptr` later when specifying the `data` variable. The struct thing is important for all structs specified by the Embedder API (I realize it doesn't matter for this particular struct which is a tagged union but it is just a good habit to get into IMO).

Reply:
Good catch, thanks.